OnlTools/Jevp/StJevpBuilders/fstBuilder.cxx

```diff
@@ -1314,37 +1356,54 @@ void fstBuilder::event(daqReader *rdr)
     }
   }
 
+  PCP;
   numTb = numTimeBin;        		//default: 9 timebins
+
+  //LOG("JEFF", "numbTB=%d", numTimeBin);
+
   memset( chCntDaq,  0, sizeof(chCntDaq) );
   memset( apvCntDaq, 0, sizeof(apvCntDaq) );
 
   if( !(evtCt %1000) )     LOG(DBG, "Looking at evt %d",evtCt);
 
   size_t evtSize = 0;
+  PCP;
 
   // ZS data stream
   daq_dta *ddZS = rdr->det("fst")->get("zs");
   if ( ddZS && ddZS->meta ) {
     apv_meta_t *meta = (apv_meta_t *) ddZS->meta;
 
+    PCP;
+
     for ( int r=1; r<=totRdo; r++ ) {                  //1--6 ARCs (ARM Readout Controllers)
       if ( meta->arc[r].present == 0 ) continue ;
       for ( int arm=0; arm<ArmPerRdo; arm++ ) {           //0--2 ARMs (APV Readout Modules) per ARC
 	if ( meta->arc[r].arm[arm].present == 0 ) continue ;
 	for ( int apv=0; apv<ApvRoPerArm; apv++ ) {         //0--7 & 12--19 APV chips per ARM
 	  if ( meta->arc[r].arm[arm].apv[apv].present == 0 ) continue ;
 
+	  PCP;
+
 	  int Tb = meta->arc[r].arm[arm].apv[apv].ntim;
+
+	  //LOG("JEFF", "Tb = %d",Tb);
+	  PCP;
+
```

Comment:
Looks like a debugging statement. Are these really needed in the production code?